src/btree/bt_handle.c

```diff
@@ -442,7 +442,7 @@ __btree_conf(WT_SESSION_IMPL *session, WT_CKPT *ckpt)
 	}
 
 	/* Initialize locks. */
-	__wt_rwlock_init(session, &btree->ovfl_lock);
+	WT_RET(__wt_rwlock_init(session, &btree->ovfl_lock));
```

Comment:
I've made them consistent with what we do elsewhere, let's follow up on that discussion outside this change.